src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -46,11 +47,53 @@ const MutationCollapsible = ({
   modifyCancerTypeModalStore,
   modifyTherapyModalStore,
   drugList,
+  handleFirebaseUpdateUntemplated,
+  handleFirebaseDeleteFromArray,
+  account,
 }: IMutationCollapsibleProps) => {
   const title = getMutationName(mutation);
   const mutationFirebasePath = buildFirebaseGenePath(hugoSymbol, `mutations/${firebaseIndex}`);
   const showMutationLevelSummary = !title.includes(',');
 
+  async function handleCreateComment(path: string, content: string, currentCommentsLength: number) {
```

Comment:
Is this the same as the one in CurationPage?

Reply:
Yeah it's the same but not sure if we wanted to couple the behavior. Figured it might be safer to have 2 copies, in case either needed additional logic, but I can pass down from the Curation Page if you prefer